### PR TITLE
Remove Normal mode

### PR DESCRIPTION
**Pull request by D-Alessian**
linked to https://github.com/Horde-Of-Greg/HOGifactory/pull/11

Needed to remove at the very least packmode as a dependency

### CODE REVIEW

**Comment by D-Alessian** [comment]
dont merge yet, we need to test first

**Comment by D-Alessian** [comment]
well it worked

File: src/main/java/com/nomiceu/nomilabs/gregtech/metatileentity/multiblock/MetaTileEntityCreativeTankProvider.java
Comment (Mireole) [inline src/main/java/com/nomiceu/nomilabs/gregtech/metatileentity/multiblock/MetaTileEntityCreativeTankProvider.java:103]: Isn't the tank provider completely useless in hm ? This just hides it from JEI, we could maybe totally get rid of it ? Same thing for the other nm-only multis from labs

File: src/main/java/com/nomiceu/nomilabs/mixin/vanilla/DedicatedServerMixin.java
Comment (Mireole) [inline src/main/java/com/nomiceu/nomilabs/mixin/vanilla/DedicatedServerMixin.java:69]: To my understanding this just makes it so that {mode} is replaced by Hog in server MOTDs. IIRC the {mode} was removed from the default MOTD in Hogifactory, so i'm not sure it still needs to exist here.

File: src/main/java/com/nomiceu/nomilabs/mixin/vanilla/DedicatedServerMixin.java
Comment (Mireole) [inline src/main/java/com/nomiceu/nomilabs/mixin/vanilla/DedicatedServerMixin.java:86]: "This is because you are on HOGIFACTORY!" -> "This is because HOGIFACTORY forces peaceful mode" or something similar ? The original sentence only really made sense with modes imo

File: src/main/java/com/nomiceu/nomilabs/mixin/vanilla/MinecraftMixin.java
Comment (Mireole) [inline src/main/java/com/nomiceu/nomilabs/mixin/vanilla/MinecraftMixin.java:52]: Same thing as with the MOTD, is this really useful ?

**Review by Mireole — CHANGES_REQUESTED** [review]

**Review by Mireole — APPROVED** [review]
All good!

**Comment by Mireole** [comment]
Please apply spotless before merging tho

**Pull request merged**
